_documentation_/cwl-vs-wdl.md

```diff
@@ -1,45 +1,71 @@
+
+## ld-pruning-wf.wdl
+* The twice-localized workaround duplicates a series of intermediate files (specifically in the merge_gds task). **The size of these intermediate files scale with the size of your input GDS files.** However, they have already been pruned by a previous task, so you can assume they will be smaller than you input files.
 * The CWL appears to contain a bug where `exclude_PCA_cor` being set to false is not respected ([#14](https://github.com/DataBiosphere/analysis_pipeline_WDL/issues/14)). The WDL avoids this. 
 * The output of merge_gds across the CWL and WDL do not md5 to the same value, but should be functionally equivalent. See [#22](https://github.com/DataBiosphere/analysis_pipeline_WDL/issues/22) for more information.   
 
-### null-model-wf.wdl
-The CWL includes a function which appears designed to have an output phenotype file inherit metadata from the required input phenotype file. It appears to either be non-functional or SB specific and therefore has not been included. We have tested the workflow extensively and have not found a situation where the phenotype output file from the WDL varies from what the phenotype output from the CWL is; ie, in spite of this deletion the two outputs md5 across workflows.
+## null-model-wf.wdl
+The majority of input files are affected by the twice-localized workaround in both tasks. As such, we recommend allocating about double the amount of disk space as the size of your inputs.
+
+The CWL includes a function which is designed to have an output phenotype file inherit metadata from the required input phenotype file. It is specific to the Seven Bridges platform and therefore has not been included in the WDL. We have tested the workflow extensively and have not found a situation where the phenotype output file from the WDL varies from what the phenotype output from the CWL is; ie, in spite of this deletion the two outputs md5 across workflows.
+
+Be aware that the original CWL gives different output for null_model_file depending on whether the Seven Bridges backend selected at workspace creation time is AWS (default) or Google. To clarify:
+
+|                    	| WDL, local 	| WDL, Terra 	| CWL, 7B via AWS 	| CWL, 7B via Google 	|
+|--------------------	|------------	|------------	|-----------------	|--------------------	|
+| WDL, local         	| pass       	| pass       	| fail            	| pass               	|
+| WDL, Terra         	| pass       	| pass       	| fail            	| pass               	|
+| CWL, 7B via AWS    	| fail       	| fail       	| pass            	| fail               	|
+| CWL, 7B via Google 	| pass       	| pass       	| fail            	| pass               	|
```

Comment:
You may want to switch from pass/fail to match/not match. You might also want to explain what is failing and how (MD5).

Reply:
This section doesn't refer to MD5 at all, so I definitely need to clarify it.